llama_hub/library.json

```diff
@@ -1074,5 +1074,14 @@
       "microsoft 365",
       "microsoft365"
     ]
+  },
+  "TrafilaturaWebReader": {
+    "id": "trafilatura_web_reader",
```

Comment:
the id should follow the name path web/trafilatura_web

Reply:
fixed now.